.changeset/rich-pigs-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react": minor
```

Comment:
This is a patch change, you can simply change the string in this file and commit it and it'll get picked up

Reply:
I was actually thinking I could make this a major change since it's not backward compatible and any users using the `useClaimedNFTs` hook will get a different behavior now

Reply:
Synced offline. Since there's no behavior change we'll leave it at patch